src-docs/src/views/elastic_charts/goal.js

```diff
@@ -1,18 +1,25 @@
-import React, { Fragment } from 'react';
+import React from 'react';
 import { Chart, Settings, Goal } from '@elastic/charts';
 import { EuiSpacer, EuiTitle, EuiCode } from '../../../../src/components';
-import { htmlIdGenerator } from '../../../../src/services';
-import { EuiFlexGroup, EuiFlexItem } from '../../../../src/components/flex';
+import {
+  htmlIdGenerator,
+  useIsWithinBreakpoints,
+} from '../../../../src/services';
+import { EuiFlexGrid, EuiFlexItem } from '../../../../src/components/flex';
 
 export const GoalChart = () => {
   const id = htmlIdGenerator()();
-  const bandLabels = ['freezing', 'chilly', 'brisk'];
-  const bands = [200, 250, 300];
+  // const themeContext = useContext(ThemeContext);
```

Comment:
TODO The goal chart is basically invisible in a dark theme.

Use ThemeContext to set a different version up